### PR TITLE
Utils proper return code

### PR DESCRIPTION
**Pull request by selvakumaar5496**
Signed-off-by: Selvakumar <selva.k.nambi@seagate.com>

# Problem Statement
- Problem statement 

# Design
-  For Bug describe the fix here. 
-  For Feature, Post the link to the solution page on the confluence CORTX Foundation Library 

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: 
-  Confirm All CODACY errors are resolved [Y/N]: 

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [ ] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [ ] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [ ] Jira is updated
- [ ] Check if the description is clear and explained. 
- [ ] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/src/setup/utils.py [hunk 1]
```diff
@@ -184,6 +183,7 @@ def validate(phase: str):
     def post_install(post_install_template: str):
         """ Performs post install operations """
 
+        rc = 0
```
Comment (sachinpunadikar) [inline py-utils/src/setup/utils.py:186]: err = None should be added

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
A small change reqd

File: py-utils/src/setup/utils.py [hunk 2]
```diff
@@ -347,24 +360,27 @@ def reset():
...
                 reset Message Bus. %s", e)
         # Clear the logs
         utils_log_path = CortxConf.get_log_path()
         if os.path.exists(utils_log_path):
             cmd = "find %s -type f -name '*.log' -exec truncate -s 0 {} +" % utils_log_path
             cmd_proc = SimpleProcess(cmd)
-            _, stderr, rc = cmd_proc.run()
-            if rc != 0:
+            _, stderr, ret_code = cmd_proc.run()
+            if ret_code != 0:
```
Comment (sachinpunadikar) [inline py-utils/src/setup/utils.py:375]: make rc =-1 inside the if block

File: py-utils/src/setup/utils_setup.py [hunk 3]
```diff
@@ -238,7 +238,7 @@ def main():
         desc = "CORTX Utils Setup command"
         Log.info(f"Starting utils_setup {argv[1]} ")
         command = Cmd.get_command(desc, argv[1:])
-        rc = command.process()
+        rc, err = command.process()
```
Comment (sachinpunadikar) [inline py-utils/src/setup/utils_setup.py:241]: remove err

**Comment by selvakumaar5496** [comment]
Closing this as it is not required anymore

**Comment by rahul-tripathi-git** [comment]
> Closing this as it is not required anymore

what was the alternate to rhis?

**Pull request closed without merging**